recipes/pymoo/LICENSE

```diff
@@ -0,0 +1,191 @@
+
```

Comment:
The license is already included in the source tarball so you don't need to (and shouldn't) include it in the recipe. Please go ahead and remove.

Reply:
That is good to know, changes have been made. Thank you!